travis/build-wheels.sh

```diff
@@ -28,22 +29,15 @@ ls -al /opt/python/
 PACKAGES="intbitset \
 bitarray \
 simplejson \
-tinyarray \
 pyahocorasick \
-billiard \
-psutil \
 requests \
 pycryptodome \
 lxml \
-url \
-url==0.1.6 \
+urlpy \
 markupsafe \
 scandir \
 pyyaml==3.12 \
-pyyaml \
-pyroaring \
-edlib \
-cdifflib 
+pyyaml  
```

Comment:
can you pin the versions? Also this is already in as pyyaml==3.12 above so this may not be needed?